### DISCUSSION TITLE
关于周本武器天赋的想法

### DISCUSSION THREAD

**Discussion by Arondight** [Arondight/Adachi-BOT#253] [Ideas]
当前是直接扒了米游社的图，爽是挺爽，但是一旦这个作者停更了并且没有其他合适的更新图就很麻烦（这个大小合适，有的太大了几十艾亩）……

解决的办法有以下几个：

1. 直接无头浏览器渲染三张图，这个维护起来简单只需要写好配置文件即可，但是性能低并且当前我们根本没有这么做的精力和技术能力，直接否掉
2. 也找个地方画个图，可以扔在阿里云 oss 上，当成三方 api 使用，但是这个也有不少劳动在里面，而且维护成本很高，并且得自己保证 api 稳定
3. 改成文字，因为过长，需要改成几个功能，今天、明天、昨天、周几后跟天赋、武器，维护简单，但是周本太过冗长，另外有时候需要好几个回复才能拿到自己想要的信息，容易刷屏，并且得到的回复不像一个总图一样全群的人都能看，一个人问了往往第二个也要问
4. 老天保佑米游社总有人去做一个大小合适的汇总图（我最倾向这个），本地缓存性能够好，够清晰，无任何维护成本，不花任何精力

至少当前没问题，那就先不改，出了问题也有一个版本的时间去改，但是怎么改到时候会是个问题

**Comment by mark9804** [comment]
米游社创作者的背后一般是组织或者团体，这个应该不用太担心

最坏的情况下可以拿excel画三张表，然后扔上图床

**Comment by Arondight** [comment]
> 拿excel画三张表

我喜欢这个
